fastserde/avro-fastserde/build.gradle

```diff
@@ -8,6 +8,35 @@ plugins {
   id "java-library"
 }
 
+//This block is only supported and required when building with JDK11
+if (JavaVersion.current() >= JavaVersion.VERSION_11) {
+  sourceSets {
+    java11 {
+      java {
+        srcDirs = ['src/main/java11']
+      }
+    }
+  }
+
+  compileJava11Java {
```

Comment:
nit - you should add a comment saying this task is auto-generated by the java plugin for your source set (otherwise it looks unused)